### PR TITLE
[KissLove] add website support (#1658)

### LINKED ISSUE #1658: [Site Request] kisslove (Raw)
Site; KissLove
SiteUrl; https://kisslove.net
SiteMangaList; https://kisslove.net/manga-list.html
Site; RAW MANGA
Site language; Japannese

Their themes look like; loveheaven.net

### PR DESCRIPTION
**Pull request by dodieboy**

### CODE REVIEW

File: src/web/mjs/connectors/KissLove.mjs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import FlatManga from './templates/FlatManga.mjs';
+
+export default class LHScan extends FlatManga {
```
Comment (27136773) [inline src/web/mjs/connectors/KissLove.mjs]: Change classname from `LHScan` to `KissLove`, i think there is already a class with that name.

File: src/web/mjs/connectors/KissLove.mjs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+    constructor() {
+        super();
+        super.id = 'kisslove';
+        super.label = 'KissLove';
+        this.tags = [ 'manga', 'raw', 'japanese' ];
+        this.url = 'https://kisslove.net/';
```
Comment (27136773) [inline src/web/mjs/connectors/KissLove.mjs]: May remove trailing slash

**Review by 27136773 — COMMENTED** [review]
Looks good, but classname collision needs to be addressed before merge

**Comment by XenonR** [comment]
Would close #1658

File: src/web/mjs/connectors/KissLove.mjs (hunk 2 above)
Comment (dodieboy) [reply src/web/mjs/connectors/KissLove.mjs]: omg why i keep forgetting to remove slash. fixed that alr, sry

**Review by 27136773 — APPROVED** [review]
Looks good

**Pull request merged**